test/hybig/HyBIG_Regression.ipynb

```diff
@@ -0,0 +1,467 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "32af63de",
+   "metadata": {},
+   "source": [
+    "# Harmony Browse Image Service (HyBIG) regression tests\n",
+    "\n",
+    "This Jupyter notebook runs a suite of regression tests against some requests against the Harmony Browse Image Service.\n",
+    "\n",
+    "These tests use ASTER Global Digital Elevation Model (GDEM) Version 3 ([ASTGTM](https://cmr.uat.earthdata.nasa.gov/search/concepts/C1256584478-EEDTEST)) as GeoTIFF input data to test the HyBIG service for an image with no color information.\n",
+    "\n",
+    "These tests use MEaSUREs Vegetation Continuous Fields (VCF) Yearly Global 0.05 Deg V001 ([VCF5KYR](https://cmr.uat.earthdata.nasa.gov/search/concepts/C1258119317-EEDTEST)) as GeoTIFF input data to test the HyBIG service against RGB color banded input.\n",
+    "\n",
+    "## Set the Harmony environment:\n",
+    "\n",
+    "The cell below sets the `harmony_host_url` to one of the following valid values:\n",
+    "\n",
+    "* Production: <https://harmony.earthdata.nasa.gov>\n",
+    "* UAT: <https://harmony.uat.earthdata.nasa.gov>\n",
+    "* SIT: <https://harmony.sit.earthdata.nasa.gov>\n",
+    "* Local: <http://localhost:3000>\n",
+    "\n",
+    "The default value is for the UAT environment. When using this notebook there are two ways to use the non-default environment:\n",
+    "\n",
+    "* Run this notebook in a local Jupyter notebook server and change the value of `harmony_host_url` in the cell below to the value for the environment you require from the above list.\n",
+    "\n",
+    "* Use the `run_notebooks.sh` script, which requires you to declare an environment variable `HARMONY_HOST_URL`. Set that environment variable to the value above that corresponds to the environment you want to test. That environment variable will take precedence over the default value in the cell below."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "dec3bc66",
+   "metadata": {
+    "tags": [
+     "parameters"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "harmony_host_url = 'https://harmony.sit.earthdata.nasa.gov'"
```

Comment:
Nit: In the description above, you mention the default is UAT (which sounds right to me), but this is SIT.

Reply:
efad46c

Reply:
efad46c